### ISSUE TITLE
Unable to log-in after applying the "require password for learners" setting

### ISSUE THREAD

**Issue by lauradanforth** [learningequality/kolibri#8777]
## Observed behavior
Under the admin facility settings, I toggled on the option to require learners to enter a password when logging in. I then logged out, and typed in my existing learner username.

After clicking "Next", instead of showing the password field, it displayed this button to create an account.

![image](https://user-images.githubusercontent.com/22602993/143102555-6224b574-9ac0-4ff9-a26e-8d81db8c9d92.png)

Then when I tried to re-create that learner account by typing in the same username into the account creation field, it indicated that the account already existed.

…

## Errors and logs

```
Failed to load resource: the server responded with a status of 400 (Bad Request)
vue-router.esm.js:2062 Uncaught (in promise) Error: Avoided redundant navigation to current location: "/create_account?step=1".
    at createRouterError (vue-router.esm.js:2062)
    at HashHistory.History.confirmTransition (vue-router.esm.js:2035)
    at HashHistory.History.transitionTo (vue-router.esm.js:2125)
    at HashHistory.replace (vue-router.esm.js:2611)
    at vue-router.esm.js:2926
    at new Promise (<anonymous>)
    at VueRouter.replace (vue-router.esm.js:2925)
    at Sub.goToFirstStep (SignUpPage.vue:213)
    at Sub.beforeMount (SignUpPage.vue:189)
    at invokeWithErrorHandling (vue.runtime.esm.js:1863)
:8080/api/auth/session/:1 Failed to load resource: the server responded with a status of 400 (Bad Request)
:8080/api/auth/session/:1 Failed to load resource: the server responded with a status of 400 (Bad Request)
vue-router.esm.js:2062 Uncaught (in promise) Error: Avoided redundant navigation to current location: "/signin?username=learner&password=&facility=426b127df141c13d76845bfcf9a1d4da".
    at createRouterError (vue-router.esm.js:2062)
    at HashHistory.History.confirmTransition (vue-router.esm.js:2035)
    at HashHistory.History.transitionTo (vue-router.esm.js:2125)
    at HashHistory.push (vue-router.esm.js:2595)
    at vue-router.esm.js:2913
    at new Promise (<anonymous>)
    at VueRouter.push (vue-router.esm.js:2912)
    at index.vue:485
```
-->

…

## Context

Tell us about your environment, including:
 * Kolibri version 0.15 beta 2
 * Operating system MacOS (using pex)
 * Browser Chrome

…

**Comment by radinamatic** [comment]
This most probably has the same root cause as #8687, and has been fixed in #8699.

**Comment by rtibbles** [comment]
Presumed fixed by #8699
